versioned_sidebars/version-2022sp-sidebars.json

```diff
@@ -0,0 +1,90 @@
+{
+  "docs": {
+    "Getting Started": [
+      {
+        "type": "doc",
+        "id": "version-2022sp/introduction"
+      },
+      {
+        "type": "doc",
+        "id": "version-2022sp/setup-environment"
+      },
+      {
+        "type": "doc",
+        "id": "version-2022sp/setup-editor"
+      }
+    ],
+    "Lectures": [
+      {
+        "type": "doc",
+        "id": "version-2022sp/lecture1"
```

Comment:
Let's leave out broken links like these

Reply:
removed broken links to lectures, assignments